### DISCUSSION TITLE
Trying to parse a Parameters resource from querying the FHIR LOINC server

### DISCUSSION THREAD

**Discussion by anibal2j** [nazrulworld/fhir.resources#122] [General]
Hi there,

Kind of new to FHIR here.
I'm able to query the LOINC FHIR server with plain python requests:

```
    headers = {'Content-Type': 'application/fhir+json'}
    url = 'https://fhir.loinc.org/CodeSystem/$lookup?system=http://loinc.org&code=4544-3'
    response = requests.get(url, auth=HTTPBasicAuth(username, password), headers=headers)
    print(f"Got response: {response.status_code}")

```

The above works and if I print out response.text I get a nice JSON payload. I'd like to be able to parse the response without having to build my own parser. It seems fhir.resources contains fhir.resources.parameters but I can't find any documentation on how to access it, how do I pass the output I get from my request above, and then how do I navigate the actual parsed response.

Any pointers?

**Comment by kpuscas** [comment]
All you should need to do is parse the json into a Parameters object. Something like

Parameters.parse_obj(params)
